services/QuillLMS/client/app/bundles/Teacher/components/dashboard/activity_feed.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import { requestGet } from '../../../../modules/request';
+
+const listIllustrationSrc = `${process.env.CDN_URL}/images/pages/dashboard/illustrations-list.svg`
```

Comment:
think we want this with the rest in the Shared directory

Reply:
I have only been putting icons in the shared directory - I think for more specific illustrations like this, that are unlikely to be shared, it makes sense to keep those in the file where they're being used so it doesn't get too cluttered.